readme.txt

```diff
@@ -6,7 +6,7 @@ License: GPLv2 or later (of-course)
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Requires at least: 3.0
 Tested up to: 6.1
```

Comment:
Missed updating Tested up to. Please makesure that fields in readme.md and readme.txt are similar.